apps/src/lib/kits/maker/Thermometer.js

```diff
@@ -31,4 +32,60 @@ const Thermometer = {
     }
   }
 };
-export default Thermometer;
+export default PlaygroundThermometer;
+
+export class MicroBitThermometer extends EventEmitter {
+  constructor(board) {
+    super();
+    this.currentTemp = 0;
+    this.board = board;
+    this.board.mb.addFirmataUpdateListener(() => {
+      this.emit('data');
+
+      // If the temp value has changed, update the local value and
+      // trigger a change event
+      if (this.currentTemp !== this.board.mb.analogChannel[12]) {
+        this.currentTemp = this.board.mb.analogChannel[12];
+        this.emit('change');
+      }
+    });
+    this.start();
+
+    Object.defineProperties(this, {
+      raw: {
+        get: function() {
+          return this.board.mb.analogChannel[12];
+        }
+      },
+      celsius: {
+        get: function() {
+          return this.raw;
+        }
+      },
+      fahrenheit: {
+        get: function() {
+          let rawValue = (this.celsius * 9) / 5 + 32;
+          return Math.floor(rawValue * 100) / 100;
```

Comment:
I noticed you used a `roundToHundreth` function in your Accelerometer PR. Seems like a handy function for a lot of MicroBit work. Mind making it a util function and using it here too?